### PR TITLE
Update Tags

### PR DESCRIPTION
**Pull request by erfan-goodarzi**
## Description

This PR adds Tags and completes some video tags.

### CODE REVIEW

File: src/data/tags.ts [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+  'Software Engineer',
+  'Project Management',
+  'Software Planning',
+  'Solution Architecture',
+  'Software Architecture',
+  'IT Strategy',
+  'Interview Tips',
+  'Team lead',
+  'HR',
+  'Tech recruiter',
```
Comment (p3yman) [inline src/data/tags.ts:25]: Tech Recruiter

File: src/data/tags.ts [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+  'C#',
+  '.Net',
+  'SRE',
+  'DevOps',
+  'IT Operations',
+  'Scalability',
+  'Software Developer',
+  'Software Engineer',
+  'Project Management',
+  'Software Planning',
```
Comment (p3yman) [inline src/data/tags.ts:18]: What is Software Planning? Is this mentioned somewhere?

**Review by p3yman — CHANGES_REQUESTED** [review]
Awesome job Erfan. Just a few comments

File: src/data/tags.ts (hunk 2 above)
Comment (erfan-goodarzi) [reply src/data/tags.ts:18]: I just added related tags. We can remove it. this is no big deal.

Comment (p3yman) [reply src/data/tags.ts:18]: I don't know what is this tag. Please remove it.

**Review by p3yman — CHANGES_REQUESTED** [review]

**Review by p3yman — APPROVED** [review]

**Pull request merged**